homeassistant/components/overseerr/diagnostics.py

```diff
@@ -18,9 +18,9 @@ async def async_get_config_entry_diagnostics(
 
     has_cloudhooks = CONF_CLOUDHOOK_URL in entry.data
 
-    data = entry.runtime_data
+    data: OverseerrData = entry.runtime_data.data
```

Comment:
```suggestion
    data = entry.runtime_data.data
```
Shouldn't be needed

Reply:
It's nice to have things typed, but I agree not useful here.

Reply:
Oh but the catch is that the `entry` already is typed, so we're able to find out what the type of `entry.runtime_data.data` is without explicitly defining it :)